src/landing.tsx

```diff
@@ -0,0 +1,177 @@
+              <img src={rLogo} alt="R logo" />
+            </a>
+          </div>
+          <a
+            href="#"
+            className="je-upload"
+            onClick={e => {
+              e.preventDefault();
+              handleUpload();
+            }}
+          >
+            Upload a Notebook
+          </a>
+        </main>
+
+        <div className="je-hero-bottom">
+          <a href="#features" className="je-scroll-indicator">
+            Scroll to learn more
+            <br />
+            <img src={scrolldownArrow} alt="" aria-hidden="true" />
+          </a>
+        </div>
+      </section>
+
+      {/* Features section */}
+      <section id="features" className="je-features">
+        <h2>Features</h2>
+        <div className="je-features-grid">
+          <div className="je-feature">
+            <img src={codeSymbol} alt="Code symbol" />
+            <h3>Code Ready</h3>
+            <p>
+              You can choose between the Python and R languages to create classroom friendly
+              interactive notebooks
+            </p>
+          </div>
+
+          <div className="je-feature">
+            <img src={shareIcon} alt="Share icon" />
+            <h3>Shareable</h3>
+            <p>
+              You can easily generate a shareable link that will allow you to share your work with
+              others or come back to it later
+            </p>
+          </div>
+
+          <div className="je-feature">
+            <img src={jupyterLogo} alt="Jupyter logo" />
+            <h3>Jupyter Compatible</h3>
+            <p>Notebooks are compatible with the JupyterLab and Jupyter notebooks ecosystems</p>
+          </div>
+        </div>
+      </section>
+
+      {/* Testimonial section */}
+      <section className="je-testimonial">
+        <img src={testimonialAvatar} alt="User avatar" className="je-avatar" />
+        <blockquote>
+          “This application makes it so much easier for us to focus on the lesson and not on
+          technical issues. I have already recommended it to others.”
+        </blockquote>
+      </section>
+
+      <section className="je-about">
+        <p>
+          Jupyter Everywhere (JE) is a collaborative project between Skew the Script and CourseKata,
+          launched in 2024 with support from the Gates Foundation. Our initiative focuses on
+          bringing data science tools and resources into classrooms by providing access to
+          high-quality tools. Our goal is to empower teachers and students to explore data science
+          and statistics easily, fostering deeper engagement and understanding in these essential
+          fields.
+        </p>
+      </section>
+
+      <section className="je-partner-logos">
+        {/* Logos aren't working... only SVGs work right now */}
+        <img src="../style/icons/landing/coursekata.png" alt="CourseKata logo" />
+        <img src="../style/icons/landing/skewthescript.jpeg" alt="Skew The Script logo" />
```

Comment:
Can you put these logos as static assets for jupyterlite? This could be in `contents` in a hidden directory or in `static` if possible